spec/jobs/whitehall_document_import_job_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.describe WhitehallDocumentImportJob do
+  include ActiveJob::TestHelper
+
+  let(:whitehall_migration) { create(:whitehall_migration) }
+
+  let(:whitehall_migration_document_import) do
+    create(:whitehall_migration_document_import, whitehall_migration_id: whitehall_migration["id"])
+  end
+
+  before do
+    allow(WhitehallImporter).to receive(:import_and_sync)
+    allow(whitehall_migration).to receive(:check_migration_finished)
```

Comment:
We mocked these since the `import_and_sync` method makes HTTP requests, which would need to be stubbed.  It didn't seem like the right place to test these requests, since they are being tested elsewhere.